src/components/designSystem/Filters/ActiveFiltersList.tsx

```diff
@@ -53,22 +51,15 @@ export const ActiveFiltersList = ({ filters }: ActiveFiltersListProps) => {
   return (
     <>
       {activeFilters.map(({ label, value }, index) => (
-        <ActiveFilterChip key={`active-filter-${index}`}>
+        <div
+          className="box-border flex h-8 items-center rounded-[100px] bg-grey-100 px-3 py-0"
```

Comment:
I don't think box-border is useful here as it's applied by default by tailwind everywhere

Reply:
indeed! Removed them in all filters components and pushed